### ISSUE TITLE
Error while trying to build Docker image

### ISSUE THREAD

**Issue by shivanip14** [google-research/football#341]
I'm trying to build the Docker image on x64 Win 11 running Python 3.9 (& using venv). While trying to build the docker image, it fails with the below error:

 > [6/7] RUN cd /gfootball && python3 -m pip install .:
#10 1.708 Processing /gfootball
#10 1.710   Preparing metadata (setup.py): started
#10 2.121   Preparing metadata (setup.py): finished with status 'done'
#10 2.682 Collecting pygame>=1.9.6
#10 2.755   Downloading pygame-2.1.2-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (21.8 MB)
#10 6.146      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 21.8/21.8 MB 5.4 MB/s eta 0:00:00
#10 6.586 Collecting opencv-python
#10 6.611   Downloading opencv_python-4.7.0.68-cp37-abi3-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (61.8 MB)
#10 15.80      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 61.8/61.8 MB 4.3 MB/s eta 0:00:00
#10 16.05 Requirement already satisfied: psutil in /usr/local/lib/python3.8/dist-packages (from gfootball==2.10.3) (5.9.4)
#10 16.62 Collecting numpy
#10 16.63   Downloading numpy-1.24.1-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (17.3 MB)
#10 19.06      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 17.3/17.3 MB 6.5 MB/s eta 0:00:00
#10 19.22 Collecting gym<=0.20.0
#10 19.24   Downloading gym-0.20.0.tar.gz (1.6 MB)
#10 19.48      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.6/1.6 MB 6.6 MB/s eta 0:00:00
#10 19.77   Preparing metadata (setup.py): started
#10 19.99   Preparing metadata (setup.py): finished with status 'error'
#10 20.00   error: subprocess-exited-with-error
#10 20.00
#10 20.00   × python setup.py egg_info did not run successfully.
#10 20.00   │ exit code: 1
#10 20.00   ╰─> [1 lines of output]
#10 20.00       error in gym setup command: 'extras_require' must be a dictionary whose values are strings or lists of strings containing valid project/version requirement specifiers.
#10 20.00       [end of output]
#10 20.00
#10 20.00   note: This error originates from a subprocess, and is likely not a problem with pip.
#10 20.01 error: metadata-generation-failed
#10 20.01
#10 20.01 × Encountered error while generating package metadata.
#10 20.01 ╰─> See above for output.
#10 20.01
#10 20.01 note: This is an issue with the package mentioned above, not pip.
#10 20.01 hint: See above for details.
------
executor failed running [/bin/sh -c cd /gfootball && python3 -m pip install .]: exit code: 1

My dependencies are:

absl-py==1.4.0
astunparse==1.6.3
cachetools==5.3.0
certifi==2022.12.7
charset-normalizer==3.0.1
cloudpickle==2.2.1
flatbuffers==23.1.21
gast==0.4.0
gfootball @ file:///[mypath]/grf-yet-again/football
google-auth==2.16.0
google-auth-oauthlib==0.4.6
google-pasta==0.2.0
grpcio==1.51.1
gym==0.21.0
h5py==3.8.0
idna==3.4
importlib-metadata==6.0.0
keras==2.11.0
libclang==15.0.6.1
Markdown==3.4.1
MarkupSafe==2.1.2
numpy==1.24.1
oauthlib==3.2.2
opencv-python==4.7.0.68
opt-einsum==3.3.0
packaging==23.0
pip==23.0
protobuf==3.19.6
psutil==5.4.3
pyasn1==0.4.8
pyasn1-modules==0.2.8
pygame==2.1.2
requests==2.28.2
requests-oauthlib==1.3.1
rsa==4.9
setuptools==65.5.0
six==1.16.0
tensorboard==2.11.2
tensorboard-data-server==0.6.1
tensorboard-plugin-wit==1.8.1
tensorflow==2.11.0
tensorflow-estimator==2.11.0
tensorflow-intel==2.11.0
tensorflow-io-gcs-filesystem==0.30.0
termcolor==2.2.0
typing_extensions==4.4.0
urllib3==1.26.14
Werkzeug==2.2.2
wheel==0.38.4
wrapt==1.14.1
zipp==3.12.0

I tried to downgrade psutil from 5.9.4 to 5.4.3, and setuptools from 67.0.0 to 65.5.0, but the error still persists.

**Comment by LINGYUNFDU** [comment]
Any updates on this issue? I have the same problem.
